### ISSUE TITLE
external library and typings

### ISSUE THREAD

**Issue by ericlormul** [mgechev/angular2-seed#820]
Hi, I want to use pickadate library in my project. I did following steps.
1. npm install pickadate --save
2. typings install pickadate --ambient --save
3. include these
      {src: 'pickadate/lib/picker.js', inject: 'libs'},
      {src: 'pickadate/lib/picker.date.js', inject: 'libs'},
      {src: 'pickadate/lib/themes/default.css', inject: true},
      {src: 'pickadate/lib/themes/default.date.css', inject: true}
   in project.config.ts

In typings/main/ambient/pickadate folder, index.d.ts file does exist. I use npm start to run the application. It runs well and pickadate works well too. The only issue is I still got the error:
error TS2339: Property 'pickadate' does not exist on type 'ElementFinder'.
on the file in which I do "$('.datepicker').pickadate();"

I did the same steps for another external library 'ckeditor'. It's fine without error.
Another question is why I don't see definition file for jquery, but $ works well without error in typescript files...

Thanks in advance for your time reading through this.

**Comment by ludohenin** [comment]
> error TS2339: Property 'pickadate' does not exist on type 'ElementFinder'.

The problem comes from the definition file you have installed (it doesn't extend jQuery typings properly), you should contact the author.

> Another question is why I don't see definition file for jquery, but $ works well without error in typescript files...

Have you installed jQuery def. file ?

**Comment by ericlormul** [comment]
Hi ludohenin,
Thank you so much for your reply!
Maybe that's because I miss jquery definition files. I haven't  manually install jQuery definition file. I just include this line
'{src: 'jquery/dist/jquery.min.js', inject: 'libs'}' in tools/config/project.config.ts, then $ works. I thought the definition comes with angular2-seed. 

I have tried once to install jQuery by 'typings install jquery ', but after installation. some error pops up and the application can't work. then I removed it.
